intel_extension_for_transformers/llm/runtime/graph/models/llama/llama_utils.cpp

```diff
@@ -132,8 +133,15 @@ void Llama::load(model_context& lctx, model_progress_callback progress_callback,
 
     // qkv GEMM
     layer.attn[0] = ml->get_tensor(layers_i + ".attention.wq.weight", {n_embd, n_embd}, backend);
-    layer.attn[1] = ml->get_tensor(layers_i + ".attention.wk.weight", {n_embd, n_embd}, backend);
-    layer.attn[2] = ml->get_tensor(layers_i + ".attention.wv.weight", {n_embd, n_embd}, backend);
+    if (n_head_kv == 8) {
+      n_ff = 28672;
```

Comment:
`n_ff` can be got by converting script? I remember `ChatGLM` pr introducing this var.

Reply:
fixed